src/main/java/core/basesyntax/service/impl/ConvertDataFromFileImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ConvertDataFromFile;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConvertDataFromFileImpl implements ConvertDataFromFile {
+    private static final int FRUIT_STRATEGY_POSITION = 0;
+    private static final int FRUIT_NAME_POSITION = 1;
+    private static final int FRUIT_AMOUNT_POSITION = 2;
+
+    @Override
+    public List<FruitTransaction> convertDataFromFile(List<String> inputList) {
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+        for (int i = 1; i < inputList.size(); i++) {
+            String[] splitInputList = inputList.get(i).split(",");
```

Comment:
```suggestion
            String[] splitInputList = inputList.get(i).split(SEPARATOR);
```
not required, but recommended